code/modules/mob/living/carbon/xenomorph/castes/Queen.dm

```diff
@@ -420,6 +420,8 @@
 /mob/living/carbon/xenomorph/queen/proc/check_block(mob/queen, turf/new_loc)
 	SIGNAL_HANDLER
 	for(var/mob/living/carbon/xenomorph/xeno in new_loc.contents)
+		if(xeno.pass_flags.flags_pass & PASS_MOB_THRU_XENO|PASS_MOB_THRU &! xeno.flags_pass_temp & PASS_MOB_THRU)
```

Comment:
these binary operators look real weird, is `&! xeno.flags_pass_temp & PASS_MOB_THRU` meant to be resolving as `&& !(xeno.flags_pass_temp & PASS_MOB_THRU)`?